tox.ini

```diff
@@ -8,15 +8,11 @@ setenv=
     PYTHONPATH = {toxinidir}:{env:PYTHONPATH:}
```

Comment:
Let's not have merge commits in a single PR if avoidable. Can we rebase so that this merge commit isn't necessary?